dfastmi/kernel/core.py

```diff
@@ -244,10 +212,9 @@ def estimate_sedimentation_length2(
     L : float
         The expected yearly impacted sedimentation length [m].
     """
-    Lt = [tmi[i] * celerity[i] for i in range(len(tmi))]
+    yearly_sedimentation_lengths = [tmi[i] * celerity[i] for i in range(len(tmi))]
```

Comment:
The term `yearly_sedimentation_lengths` isn't appropriate. The variable represents the contribution of each period/discharge level to the total sedimentation length, so let's use `sedimentation_length_contributions` instead.

Reply:
I will change this.